test/serializer/optimized-functions/ObjectAssign.js

```diff
@@ -1,4 +1,4 @@
-// Copies of assign\(:1
+// Copies of .assign;:1
```

Comment:
What does this test? Why is it important to have `Object.assign;` in the output?

Reply:
So we know it hasn't incorrectly been removed.